content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -60,13 +62,23 @@ components, most are enabled by default:
   gate](/docs/reference/command-line-tools-reference/feature-gates/) on
   kube-proxy.
 
+
+## API Fields
```

Comment:
(nit)
```suggestion
## API fields
```

Reply:
Wouldn't this be title-case, `API Fields`?

Reply:
Use sentence case for heading per https://kubernetes.io/docs/contribute/style/style-guide/#headings 

Not a deal-breaker though.

Reply:
Updated, thanks!